share/lib/python/neuron/rxd/node.py

```diff
@@ -7,7 +7,13 @@
 from .rxdException import RxDException
 import warnings
 import ctypes
-import collections
+
+# For 8.0.x release, keep backward compat with older python versions
```

Comment:
I would drop this change and prioritise `master` being clean. `collections.abc` was introduced in Python 3.3.

Reply:
This chunk of code is already in another file (I copied from there). Looking for consistency for future refactoring.

Reply:
Still seems like a bit of a backwards step to me, but up to you.

Reply:
I will actually refactor it now, only used in 2 places.